unified-codes/apps/web/src/reducers/DetailsReducer.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  DETAILS_ACTIONS,
+  IDetailsAction,
+  IDetailsFetchSuccessAction,
+  IDetailsFetchFailureAction,
+} from '../actions';
+import { IDetailsState } from '../types';
+
+const initialState = (): IDetailsState => {
+  return {
+    loading: false
+  };
+};
+
+export const DetailsReducer = (
+  state = initialState(),
+  action: IDetailsAction
+): IDetailsState => {
+  const { type } = action;
+
+  switch (type) {
+    case DETAILS_ACTIONS.FETCH_ENTITY: {
+      return { ...state, loading: true };
+    }
+    case DETAILS_ACTIONS.FETCH_SUCCESS: {
+      const { entity } = action as IDetailsFetchSuccessAction;
+      return { ...state, entity, loading: false };
```

Comment:
not a problem at the moment, and this isn't exactly going to happen. but..
I'd recommend clearing the error here as well. in case
- fetching
- fetch failure ( sets `error` )
- fetching 
- fetch success ( `error` is still set )

this is only an issue if error is used to display a problem, and that isn't happening now. nor is likely to.
and I can't see the need to fetch, fail, re-fetch. 

so, this is more abstract than useful.

Reply:
Thanks for the suggestion 😸  have applied in ca31cc6